src/firefighter/firefighter/settings/components/raid.py

```diff
@@ -17,3 +17,6 @@
 
     RAID_JIRA_USER_IDS: dict[str, str] = {}
     "Mapping of domain to default Jira user ID"
+
+    TOOLBOX_URL: str = config("TOOLBOX_URL")
```

Comment:
```suggestion
    RAID_TOOLBOX_URL: str = config("RAID_TOOLBOX_URL")
```
Settings are not namespaced, so prefixing them helps.